s2wav/directional_tiling.py

```diff
@@ -149,28 +158,34 @@ def tiling_wavelet(L: int, lam: float, spin: int, original_spin: int, N: int, J_
 
     Notes:
         [1] B. Leidstedt et. al., "S2LET: A code to perform fast wavelet analysis on the sphere", A&A, vol. 558, p. A128, 2013.
-    '''
+    """
     J = j_max(L, lam)
     el_min = max(abs(spin), abs(original_spin))
 
-    phi = np.zeros(L, dtype = np.float64)
-    psi = np.zeros((J+1) * L * L, dtype = np.complex128) 
-    kappa = np.zeros((J+1) * L, dtype = np.float64)
-    kappa0 = np.zeros(L, dtype = np.float64)
+    phi = np.zeros(L, dtype=np.float64)
+    psi = np.zeros((J + 1) * L * L, dtype=np.complex128)
+    kappa = np.zeros((J + 1) * L, dtype=np.float64)
+    kappa0 = np.zeros(L, dtype=np.float64)
     s_elm = tiling_direction(N, L)
 
     for el in range(el_min, L):
         phi[el] = math.sqrt((2 * el + 1) / 4.0 * math.pi * kappa0[el])
         if original_spin != 0:
-            phi[el] *= spin_normalization(el, original_spin) * (-1)**original_spin
+            phi[el] *= spin_normalization(el, original_spin) * (-1) ** original_spin
 
     for j in range(J_min, J):
-        ind = el_min ** 2
+        ind = el_min**2
         for el in range(el_min, L):
             for m in range(-el, el):
-                psi[j * L * L + ind] = math.sqrt((2 * el + 1) / (8.0 * math.pi * math.pi)) * kappa[j * L + el] * s_elm[ind]
+                psi[j * L * L + ind] = (
+                    math.sqrt((2 * el + 1) / (8.0 * math.pi * math.pi))
+                    * kappa[j * L + el]
+                    * s_elm[ind]
+                )
                 if original_spin != 0:
-                    psi[j * L * L + ind] *= spin_normalization(el, original_spin) * (-1)**original_spin
-
+                    psi[j * L * L + ind] *= (
+                        spin_normalization(el, original_spin) * (-1) ** original_spin
+                    )
+                ind += 1
```

Comment:
This is the only real update @JessWhitney, you just missed incrementing the indexing!